ui/app/styles/components/popup-menu.scss

```diff
@@ -112,10 +112,16 @@
 .popup-menu-content .level-left {
   flex-shrink: 1;
 }
-.popup-menu-trigger {
-  height: 2rem;
-  min-width: 0;
-  padding: 0 $size-10;
+
+.list-item-row,
+.info-table-row,
+.wizard-dismiss-menu {
+  .popup-menu-trigger {
+    height: 2.5rem;
+    min-width: 0;
+    padding: 0;
+    width: 2.5rem;
+  }
```

Comment:
I think it might be good to introduce an "Icon button" style rather than do it this way, but this is an easy reset for now